.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
 
-before_install:
-    - wget http://bit.ly/miniconda -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - conda update --yes conda
-    - travis_retry conda create --yes -n test $CONDA --file requirements.txt
-    - source activate test
-    - travis_retry conda install --yes pip
-    - travis_retry pip install pytest
+# tdk: enable master when merged
+branches:
```

Comment:
Unnecessary.  This branch can be tested as a PR and later on it will be merged, right?